packages/strapi-plugin-versioning/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "strapi-plugin-versioning",
+  "version": "0.1.0",
+  "description": "Strapi plugin for collecting versions of content type entities",
+  "main": "src/index.ts",
+  "scripts": {
+    "test": "jest --coverage",
+    "build": "tsc",
+    "clean": "rimraf dist coverage",
+    "clean-all": "rimraf dist coverage node_modules",
+    "set-development-package-json": "ts-node ../../scripts/set-development-package-json.ts",
```

Comment:
I think this is neccessary, 

in pre publish You should extract from dist, on postPublish You should cleanup 

but we talked about changes and change tsc to compile to `.` (root) and add proper structure to .gitignore